### ISSUE TITLE
C.SI.1.1 - COERENZA DELL'UTILIZZO DEI FONT (librerie di caratteri) - Il sito comunale deve utilizzare i font indicati dalla documentazione del modello di sito comunale

### ISSUE THREAD

**Issue by marco-nanni** [italia/design-comuni-wordpress-theme#357]
### Prerequisiti

- [X] Ho già [cercato](https://github.com/italia/design-comuni-wordpress-theme/issues?utf8=%E2%9C%93&q=is%3Aissue) tra le issue la presenza di un problema simile
- [X] Ho letto le [linee guida di contribuzione](https://github.com/italia/design-comuni-wordpress-theme/blob/main/CONTRIBUTING.md)

### Descrivi il problema

Avevo segnalato questo problema tempo fa senza ricevere risposta.
Ora il problema si ripresenta e tra le altre cose non in tutte le installazioni.
Abbiamno fatto tutte le verifiche del caso. 
I data-element sono presenti. Tutti.
I font sono quelli previsti e caricati correttamenete e non si evidenziano errori.
Cosa altro rimane da controllare?
A quali <h> e <p> fa riferimento esattamente?

### Includi link o screenshot

![C SI 1 1](https://github.com/italia/design-comuni-wordpress-theme/assets/118733111/6a03aa59-7379-4a93-a885-d1daf246aa42)
![font_comuni_caricati](https://github.com/italia/design-comuni-wordpress-theme/assets/118733111/7ee2534f-56b6-4cbe-98a8-c1f30ed51af7)

### Su quale sisitema operativo stai riscontrando il problema?

Linux

### Su quale browser stai riscontrando il problema?

Qualsiasi

### Che versione del tema stai utilizzando?

1.7.2

**Comment by marco-nanni** [comment]
In che senso?
In effetti è installato. Non da noi.

**Comment by enrimk** [comment]
Nel senso che vedo che ha un font stack "Helvetica, Arial, sans-serif".
Può darsi che sia quello che viene rilevato.

**Comment by marco-nanni** [comment]
Abbi pietà, sarò fuso... ma dove vedi "Helvetica, Arial, sans-serif", ma poi 42 items?
Potrebbe anche essere.
Ho disabilitato la cache, il firewall (non installati da noi)... ma nisba.
La cosa bella che in sviluppo non fa una piega e in effetti questi tre plugin non ci sono.

**Comment by marco-nanni** [comment]
Ottimo, grazie.
Ora chiedo di disabilitarlo e rifaccio i test. 
In effetti la tua domanda ha molto senso. Prevedo anch'io casini a manetta.

**Comment by zetareticoli** [comment]
@marco-nanni attendo di conoscere il risultato dei nuovi test

**Comment by zetareticoli** [comment]
Ok bene, giusto per capire se c'era un problema con l'app di valutazione o con il modello. Chiudo questa issue intanto. Per quanto riguarda i plugin, molti iniettano codice nel DOM pertanto quando fate i test potrebbe essere utile disattivarli per non avere dei falsi negativi.

**Comment by marco-nanni** [comment]
@zetareticoli certo, a sapere cosa viene installato... perché per scoprire che era quella la causa non ci sono voluti 5 minuti.
E comuqne rimane di fatto un problema. Perché quello magari lo puoi disattivare momentaneamente, altro dipende.
